SWIGGYDEMO_REACT/swiggy-react-app/src/components/itemDetailsAndCart/components/listByCategory/components/items/components/item/Item.js

```diff
@@ -0,0 +1,52 @@
+import './item.style.css';
+
+import React from "react";
+import AddButtonContainer from "./components/addButton/index";
+import ActionButtonForItemsContainer from './components/actionButtonForItems/ActionButtonForItemsContainer';
+
+function Item ({item, cartItems, onClickADD, onClickPlus, onClickMinus}) {
+
+    let flag = false;
```

Comment:
no let or for loop, use alternative solution using const and map/filter/forEach or reduce